frontend_build/src/read_bundle_plugins.js

```diff
@@ -24,8 +24,6 @@ var libs = function(kolibri_name) {
     'logging': kolibri_name + '.lib.logging',
     'vue': kolibri_name + '.lib.vue',
     'kolibri': kolibri_name,
-    'core-base': kolibri_name + '.lib.coreBase',
```

Comment:
No, dev and production work identically, the only danger is of code duplication, where code that should only be in core gets repeated across other modules.